src/main/scala/akka/contrib/process/BlockingProcess.scala

```diff
@@ -107,16 +109,11 @@ class BlockingProcess(
     }
 
     try {
-      val stdin =
-        context.actorOf(OutputStreamSubscriber.props(process.getOutputStream), "stdin")
+      val stdin = Sink.outputStream(process.getOutputStream)
+      val stdout = Source.inputStream(process.getInputStream)
+      val stderr = Source.inputStream(process.getErrorStream)
```

Comment:
Sure looks a lot nicer. 

Are the actors created by these sources still children of this actor? (they need to be) Are there any guarantees that this will be the case - even if stated in Akka API doco?